driver/configuration/provider_koanf.go

```diff
@@ -280,7 +288,44 @@ func (v *KoanfProvider) ToScopeStrategy(value string, key string) fosite.ScopeSt
 }
 
 func (v *KoanfProvider) pipelineIsEnabled(prefix, id string) bool {
-	return v.source.Bool(fmt.Sprintf("%s.%s.enabled", prefix, id))
```

Comment:
I think this is very fast as it is without caching, leave it as is :)

Reply:
I wasn't sure how long loading the `source.Bool` would take (if it's reloading something or not).

Commit dropped ✔️